lib/kb_SetUtilities/kb_SetUtilitiesImpl.py

```diff
@@ -238,21 +318,17 @@ def KButil_Localize_FeatureSet(self, ctx, params):
                 if src_genome_ref in genome_ref_to_standardized:
                     pass
                 else:
-                    try:
-                        src_genome_obj_info = wsClient.get_object_info_new ({'objects':[{'ref':src_genome_ref}]})[0]
-                        src_genome_obj_type = re.sub ('-[0-9]+\.[0-9]+$', "", src_genome_obj_info[TYPE_I])  # remove trailing version
-                    except Exception as e:
-                        raise ValueError('Unable to get genome object info from workspace: (' + str(src_genome_ref) +')' + str(e))
+                    (src_genome_obj_info,
+                     src_genome_obj_name,
+                     src_genome_obj_type) = self.get_obj_info(src_genome_ref, 'genome', full_type=True)
 
                     #acceptable_types = ["KBaseGenomes.Genome", "KBaseGenomeAnnotations.GenomeAnnotation"]
```

Comment:
rm comment

Reply:
leaving because we may need to support the KBaseGenomeAnnotations.GenomeAnnotation type again someday, and this is a clue to that information.